extensions/geode-modules/src/main/java/org/apache/geode/modules/session/catalina/DeltaSession.java

```diff
@@ -231,6 +236,11 @@ public void setOwner(Object manager) {
     }
   }
 
+  @SuppressWarnings("deprecation")
+  private void setOwnerDeprecated(DeltaSessionManager sessionManager) {
```

Comment:
I think the method name does not reflect PreferDeserializedForm. It might be better to be explicit instead of using Deprecated.